subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c

```diff
@@ -887,6 +946,7 @@ static void rp_pu_st_wait_tx_ack_phy(struct ll_conn *conn, struct proc_ctx *ctx,
 				rp_pu_complete(conn, ctx, evt, param);
 			}
 		}
+#endif /* CONFIG_BT_CENTRAL */
```

Comment:
I think CI will require an else after an else-if, even if empty.
And it must contain a comment (like /* do nothing */ or something).